rbac/management/role/definer.py

```diff
@@ -66,15 +58,7 @@ def _make_role(tenant, data):
             return role
     for access_item in access_list:
         resource_def_list = access_item.pop("resourceDefinitions", [])
-        permission, created = Permission.objects.get_or_create(**access_item)
-
-        # NOTE: after we ensure/enforce all object have a tenant_id FK, we can add tenant=tenant
-        # to the get_or_create. We cannot currently, because records without would fail the GET
-        # and would create duplicate records. This ensures we temporarily do an update if
-        # obj.tenant_id is NULL
-        if not permission.tenant:
-            permission.tenant = tenant
-            permission.save()
+        permission, created = Permission.objects.get_or_create(**access_item, tenant=tenant)
```

Comment:
A little worried here and the permission creation in a few places, the tenant for permission is always public, right?

Reply:
Currently it is, yes. Since we have `tenant_id` on `Permission`, we're always setting it to the public schema in seeds. What would be the concern?

Reply:
Oh cool! I thought we were setting some other tenants on them.